content/300-accelerate/800-examples.mdx

```diff
@@ -0,0 +1,21 @@
+---
+title: 'Examples'
+metaTitle: 'Prisma Accelerate: Examples'
+metaDescription: 'Check out ready-to-run examples for Prisma Accelerate.'
+tocDepth: 3
+---
+
+## Prisma Accelerate examples
```

Comment:
I think we should just link to the examples repo instead of a new page, what do you think?

Reply:
Yea i disagree with putting an external link in the side nav. Also I think having the examples be in the docs allows us more flexibility.

Reply:
Hmh, okay. I indeed would've preferred just an external link. What flexibility do we gain here that's valuable?